roles/neutron-data-network/tasks/lbaas-haproxy-cleanup.yml

```diff
@@ -0,0 +1,19 @@
+- name: delete neutron lbaas service (ubuntu)
+  upstart_service:
+    name: "{{ item.name }}"
+    state: absent
+    cmd: "{{ item.cmd }}"
+    user: "{{ item.user }}"
+  with_items:
+    - "{{ neutron.services.neutron_lbaasv2_agent }}"
+  when: ursula_os == 'ubuntu'
+  
+- name: delete lbaas templates folder
```

Comment:
Deleting neutron haproxy lbaasv2 service/configuration is for safety when enabling f5. It can avoid some complicated problem when having haproxy lbaasv2 and f5 lbaasv2 at the same time. The similar code applied to F5 service/configuration when enabling haproxy lbaasv2.